 * Serving Flask app 'app'
 * Debug mode: off
[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[33mPress CTRL+C to quit[0m
127.0.0.1 - - [27/Apr/2024 00:42:37] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [27/Apr/2024 00:42:38] "[33mGET /cart.json HTTP/1.1[0m" 404 -
127.0.0.1 - - [27/Apr/2024 00:42:40] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [27/Apr/2024 00:42:41] "[33mGET /cart.json HTTP/1.1[0m" 404 -
127.0.0.1 - - [27/Apr/2024 00:42:41] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
127.0.0.1 - - [27/Apr/2024 00:42:47] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [27/Apr/2024 00:42:47] "[33mGET /cart.json HTTP/1.1[0m" 404 -
127.0.0.1 - - [27/Apr/2024 00:42:54] "GET / HTTP/1.1" 200 -
Traceback (most recent call last):
  File "app.py", line 18, in <module>
    def update_task():
  File "/home/ammara/Desktop/DEVOPS/Projects/ToDoList/ToDoList/lib/python3.8/site-packages/flask/sansio/scaffold.py", line 362, in decorator
    self.add_url_rule(rule, endpoint, f, **options)
  File "/home/ammara/Desktop/DEVOPS/Projects/ToDoList/ToDoList/lib/python3.8/site-packages/flask/sansio/scaffold.py", line 47, in wrapper_func
    return f(self, *args, **kwargs)
  File "/home/ammara/Desktop/DEVOPS/Projects/ToDoList/ToDoList/lib/python3.8/site-packages/flask/sansio/app.py", line 657, in add_url_rule
    raise AssertionError(
AssertionError: View function mapping is overwriting an existing endpoint function: update_task
